src/test/java/it/rate/webapp/controllers/InterestAdminControllerIntegrationTest.java

```diff
@@ -128,8 +128,6 @@ void editUsersReturnsForbiddenForVoter() throws Exception {
         .perform(get("/interests/" + interestId + "/admin/users"))
         .andExpect(status().isForbidden())
         .andReturn();
-
-    verify(manageInterestService, times(0)).getUsersByRole(interestId);

```

Comment:
why was this removed, it is important part of the test that verifies that forbidden users really didn't get access?

Reply:
getUsersByRole method no longer exist, but indeed a verification is missing here for the new method. I will update this.